src/main/daml/Daml/Finance/Interface/Instrument/Bond/Instrument.daml

```diff
@@ -0,0 +1,43 @@
+-- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module Daml.Finance.Interface.Instrument.Bond.Instrument where
+
+import Daml.Finance.Interface.Instrument.Base.Instrument qualified as Base (I, Implementation, K, asDisclosure)
+import Daml.Finance.Interface.Util.Disclosure qualified as Disclosure (I)
+
+-- | Type synonym for `Instrument`.
+type I = Instrument
+
+-- | Type synonym for `View`.
+type V = View
+
+-- View of `Instrument`.
+data View = View
+  with
+    currency : Base.K
```

Comment:
I think we need one interface per bond, per swap etc. - otherwise, as seen in this view type we don't actually get much information out of the interface. The individual interfaces really should contain all fields for a specific instrument, such that we can eg. display them in the UI.

Reply:
To be consistent with other instruments I'd suggest to create folders for `FixedRate`, `FloatingRate`, etc. and have `Instrument.daml` and `Factory.daml` in each of them.

Reply:
I was debating whether to have one interface per instrument or per asset class. I seen we had a classification task to pick up which I thought could be used to get to the specific asset type. You're correct in that the interfaces either have nothing or a single field which isn't that useful (I originally had these interfaces expose the `Base Instrument` view so we displayed something useful at least). I'll create one interface per bond/swap/etc.

> To be consistent with other instruments I'd suggest to create folders for `FixedRate`, `FloatingRate`, etc. and have `Instrument.daml` and `Factory.daml` in each of them.

👍